_opensearch/point-in-time.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Point in Time
+nav_order: 58
+has_children: true
+has_toc: false
+---
+
+# Point in Time
+
+Point in Time (PIT) lets you run different queries against a dataset that is fixed in time. 
+
+Normally, if you run a query on an index multiple times, the same query may return different results because documents are constantly indexed, updated, and deleted. If you need to run a query against the same data, you can preserve that data's state by creating a PIT. The main use of PIT is to couple it with the `search_after` functionality for deep pagination of search results.
+
+## Paginating search results
+
+Besides PIT, there are three ways to [paginate search results]({{site.url}}{{site.baseurl}}/opensearch/search/paginate) in OpenSearch: using the Scroll API, specifying `from` and `size` parameters for your search, and using the `search_after` functionality. However, all three have limitations:
+
+- The Scroll API's search results are frozen at the moment of the request, but they are bound to a particular query. Additionally, scroll can only move forward in the search, so if a request for a page fails, the subsequent request skips that page and returns the following one.
+- If you specify the `from` and `size` parameters for your search, the search results are not frozen in time, so they may be inconsistent because of documents being indexed or deleted. The `from` and `size` feature is not recommended for deep pagination because every page request requires processing of all results and filtering them for the requested page.
+- The `search_after` search results are not frozen in time, so they may be inconsistent because of concurrent indexing or deleting documents.
```

Comment:
```suggestion
- The `search_after` search results are not frozen in time, so they may be inconsistent because of concurrent document indexing or deletion.
```